README.md

```diff
@@ -95,13 +99,23 @@ mdtf
          ├── (... supporting data for individual PODs )
 ```
 
+<<<<<<< HEAD
+The default test case uses the QBOi.EXP1.AMIP.001 sample. The GFDL.CM4.c96L32.am4g10r8 sample is only for testing the MJO Propagation and Amplitude POD. More data for additional PODs (including those still under development) are available from the FTP.
+
+You can put the observational data and model output in different locations (e.g., for space reasons) by changing the values of `OBS_DATA_ROOT` and `MODEL_DATA_ROOT` as described below in section 3.
+=======
 The default test case uses the QBOi.EXP1.AMIP.001 data. The GFDL.CM4.c96L32.am4g10r8 data is only for testing the MJO Propagation and Amplitude POD.
+>>>>>>> 1c01368579b94417af4ddb1d65e12d368d43e015
 
 You can put the observational data and model output in different locations (e.g., for space reasons) by changing the values of `OBS_DATA_ROOT` and `MODEL_DATA_ROOT` as described below in section 4.
 
 ## 2. Install the conda package manager, if needed
 
+<<<<<<< HEAD
+The MDTF framework code is written in Python 3, but supports running PODs written in a variety of scripting languages and combinations of libraries. We use [Conda](https://docs.conda.io/en/latest/), a free, open-source package manager to install and manage these dependencies. Conda is one component of the [Miniconda](https://docs.conda.io/en/latest/miniconda.html) and [Anaconda](https://www.anaconda.com/) python distribution, so having Miniconda/Anaconda is sufficient but not necessary.
```

Comment:
Note that for CPO-funded PODs, Python 3 is required?

Reply:
Fixed.